### PR TITLE
Add missing license headers for /dev files

### LINKED ISSUE #1415: Missing license headers
## Are you reporting a feature request or a bug?

Task

## Description

Various files in `dev/` are missing license headers (e.g. `pastefromword/getclipboard.html`). We should probably add them.

### PR DESCRIPTION
**Pull request by Comandeer**
## What is the purpose of this pull request?

Task

## Does your PR contain necessary tests?

All patches which change the editor code must include tests. You can always read more
on [PR testing](https://docs.ckeditor.com/ckeditor4/docs/#!/guide/dev_contributing_code-section-tests),
[how to set the testing environment](https://docs.ckeditor.com/ckeditor4/docs/#!/guide/dev_tests) and
[how to create tests](https://docs.ckeditor.com/ckeditor4/docs/#!/guide/dev_tests-section-creating-your-own-test)
in the official CKEditor documentation.

### This PR contains

- [ ] Unit tests
- [ ] Manual tests

No tests are needed.

## What changes did you make?

I've added missing license headers in `dev/` files. I omit only pure grunt configuration files.

Closes #1415.

### CODE REVIEW

File: dev/langtool/meta/ckeditor.plugin-spellcheck/meta.txt [hunk 1]
```diff
@@ -1,18 +1,21 @@
+# Copyright (c) 2003-2018, CKSource - Frederico Knabben. All rights reserved.
+# For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+
 btnIgnore = Label for the Ignore button of the Spell Check dialog window.
 btnIgnoreAll = Label for the Ignore All button of the Spell Check dialog window.
-btnReplace = 
-btnReplaceAll = 
-btnUndo =
```
Comment (jacekbogdanski) [inline dev/langtool/meta/ckeditor.plugin-spellcheck/meta.txt]: It's fine that you removed redundant spaces, but this issue is very specific and it wasn't in its description. Now I don't know if it breaks anything? I think it should have separate PR.

File: dev/langtool/meta/ckeditor.plugin-spellcheck/meta.txt [hunk 2]
```diff
@@ -1,18 +1,21 @@
...
+btnReplace =
+btnReplaceAll =
+btnUndo =
 changeTo = Label for the Change To button of the Spell Check dialog window.
 errorLoading = Error message displayed when the spellchecker application service host cannot be loaded.
-ieSpellDownload = 
-manyChanges = 
-noChanges = 
-noMispell = 
-noSuggestions =
```
Comment (jacekbogdanski) [inline dev/langtool/meta/ckeditor.plugin-spellcheck/meta.txt]: See https://github.com/ckeditor/ckeditor-dev/pull/2023/files#r192353557

File: dev/langtool/meta/ckeditor.plugin-spellcheck/meta.txt [hunk 3]
```diff
@@ -1,18 +1,21 @@
...
-noSuggestions = 
+ieSpellDownload =
+manyChanges =
+noChanges =
+noMispell =
+noSuggestions =
 notAvailable = Error message displayed when the spellchecker is not available.
 notInDic = Error message displayed when a word is not in dictionary.
-oneChange = 
-progress =
```
Comment (jacekbogdanski) [inline dev/langtool/meta/ckeditor.plugin-spellcheck/meta.txt]: See https://github.com/ckeditor/ckeditor-dev/pull/2023/files#r192353557

**Comment by Comandeer** [comment]
> I was also thinking if maybe we should add license headers to githooks files?

I didn't add license as our main `gruntfile.js` also doesn't have one. I added headers only to files containing code, not pure config.

File: dev/langtool/meta/ckeditor.plugin-spellcheck/meta.txt (hunk 1 above)
Comment (Comandeer) [reply dev/langtool/meta/ckeditor.plugin-spellcheck/meta.txt]: Actually it was automatically removed by my editor. I'll undo these changes.

**Pull request merged**